src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -210,7 +211,7 @@
                 //  allowed via '4.8.3 Arrays'
                 "ARRAY_INIT",
                 // google prefers no spaces on one side or both for these tokens
-                "GENERIC_START", "GENERIC_END", "WILDCARD_TYPE")
+                "GENERIC_START", "GENERIC_END", "WILDCARD_TYPE", "ELLIPSIS")
```

Comment:
comment for "ELLIPSIS" should be " '...' is almost same as '[]' by meaning, google style never define it clearly but in all examples there are no space between type and '...'"